packages/manager/modules/hub/src/components/tile/tile.less

```diff
@@ -0,0 +1,79 @@
+.ovh-manager-hub-tile {
```

Comment:
Could you scope it to the component ? 

```suggestion
ovh-manager-hub-tile {
```

Reply:
Sorry for misunderstanding your previous. This is done now :)